test/functional/test_esxi.py

```diff
@@ -0,0 +1,195 @@
+import unittest
+import os
+import time
+import subprocess
+import yaml
+from test import fixtures
+import re
+from infrasim import model
+from infrasim import InfraSimError
+import paramiko
+from infrasim.package_install import install_bintray_packages
+import urllib
+
+old_path = os.environ.get("PATH")
+new_path = "{}/bin:{}".format(os.environ.get("PYTHONPATH"), old_path)
+source_esxi_path = "ftp://10.62.59.23/idic/iso/esxi_img/esxi6p3-1.qcow2"
```

Comment:
It's 884M. The test costs 2 minutes in our environment. But maybe much slower for dev outside of our company. How about to put the test in Puffer CI?

Ran 3 tests in 141.152s

OK

real    2m23.739s
user    0m2.892s
sys     0m2.392s